libs/journeys/ui/src/components/TemplateView/CreateJourneyButton/CreateJourneyButton.tsx

```diff
@@ -30,7 +36,8 @@ const DynamicCopyToTeamDialog = dynamic(
 )
 
 export function CreateJourneyButton({
-  signedIn = false
+  signedIn = false,
+  displayOpenTeamDialog = true
```

Comment:
```suggestion
  displayOpenTeamDialog = false
```
Set to false by default